src/main/menus/app.js

```diff
@@ -224,6 +224,13 @@ var createTemplate = function(mainWindow, config) {
   template.push({
     label: '&Help',
     submenu: [{
+      label: `${app_name} Docs`,
```

Comment:
Proposed instead of "Mattermost Docs" just use "Learn more" to bring users to specific help information.